src/pages/en/core-concepts/astro-pages.md

```diff
@@ -50,12 +57,6 @@ import MySiteLayout from '../layouts/MySiteLayout.astro';
 
 📚 Read more about [layout components](/en/core-concepts/layouts/) in Astro.
 
-#### Modifying `<head>`
-
-Note that using a `<head>` tag works like any other HTML tag: it does not get moved to the top of the page. We recommend writing `<head>` and its contents in your top-level layout.
-
-
-
 ## Markdown Pages
 
 Astro also treats any Markdown (`.md`) files inside of `/src/pages/` as pages in your final website. These are commonly used for text-heavy pages like blog posts and documentation.
```

Comment:
```suggestion
## Markdown/MDX Pages

Astro also treats any Markdown (`.md`) and (`.mdx`) files inside of `/src/pages/` as pages in your final website. These are commonly used for text-heavy pages like blog posts and documentation.
```

Reply:
In the table of contents, I listed `.md` and `.mdx` as separate and linked out to the `MDX` integrations page. Should I combine them and link them both to this?

Reply:
I would leave the links as-is above (since this doesn't really say much!) 😅  BUT include them both here together, since they work the same way. It would be weird to have one without the other.